ci/build.yml

```diff
@@ -275,10 +275,31 @@ jobs:
           echo "Could not connect to Oracle, trying again..."
           sleep 1
         done
+
+
+        # Setting MAX_STRING_SIZE='EXTENDED' in order to enable specfying collation at column level.
+        # See 'Increasing the Maximum Size of VARCHAR2, NVARCHAR2, and RAW Columns in a PDB' at
+        # https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/MAX_STRING_SIZE.html#GUID-D424D23B-0933-425F-BC69-9C0E6724693C
+        #
+        cat <<EOT > set_max_string_size.txt
```

Comment:
technically speaking, this is "sql", so I would make it apparent in the file extension